regression/tests/common/test_html_component.py

```diff
@@ -0,0 +1,115 @@
+"""
+End to end tests for HTML Componenets
+"""
+import os
+from bok_choy.web_app_test import WebAppTest
+from regression.pages.studio.course_outline_page import (
+    CourseOutlinePageExtended
+)
+from regression.pages.studio.unit_page import UnitPageExtended
+from regression.pages.studio.login_studio import StudioLogin
+from regression.pages.studio.studio_home import DashboardPageExtended
+from regression.pages.lms.login_lms import LmsLogin
+from edxapp_acceptance.pages.lms.courseware import CoursewarePage
+from regression.tests.helpers import (
+    LoginHelper, get_course_info, get_data_id_of_component
+)
+
+from edxapp_acceptance.pages.studio.utils import add_components
+
+
+class StudioLmsHTMLTest(WebAppTest):
+    """
+    HTML Components tests that require lms verification with studio
+    """
+
+    DEMO_COURSE_USER = os.environ.get('USER_LOGIN_EMAIL')
+    DEMO_COURSE_PASSWORD = os.environ.get('USER_LOGIN_PASSWORD')
+
+    def setUp(self):
+        """
+        Initialize the page object
+        """
+        super(StudioLmsHTMLTest, self).setUp()
+        # Login to Lms first to avoid authentication problems
+        self.login_page = LmsLogin(self.browser)
+        LoginHelper.login(self.login_page)
+
+        self.unit_container_page = UnitPageExtended(
+            self.browser, None
+        )
+
+        self.studio_login_page = StudioLogin(self.browser)
+        self.studio_login_page.visit()
+        self.studio_login_page.login(self.DEMO_COURSE_USER,
+                                     self.DEMO_COURSE_PASSWORD)
+        self.studio_home_page = DashboardPageExtended(self.browser)
+
+        self.course_info = get_course_info()
+
+        self.studio_course_outline = CourseOutlinePageExtended(
+            self.browser, self.course_info['org'], self.course_info['number'],
+            self.course_info['run'])
+
+        self.lms_courseware = CoursewarePage(self.browser, self.course_info)
+
+    def test_html_components(self):
+        """
+        Verifies that user can add HTML components on Studio and LMS
+        """
+        self.studio_course_outline.visit()
+        self.studio_course_outline.add_section_with_name('Test Section 1')
```

Comment:
I wonder if it would be a good idea to append a uuid to this string to avoid any collisions between test runs.